testsuite/lattice_boltzmann/test_LBMDcoupling.py

```diff
@@ -0,0 +1,126 @@
+import espressopp
+import mpi4py.MPI as MPI
+from espressopp import Int3D
+from espressopp import Real3D
+
+import unittest
+
+runSteps = 500
+temperature = 1.0
+num_particles = 400
+Ni = 10
+initDen = 1.
+initVel = 0.
+
+class TestLBMDCoupling(unittest.TestCase):
+    def setUp(self):
+        # set up system
+        global Ni, temperature
+
+        box  = (Ni, Ni, Ni)
+        rc   = pow(2, 1./6.)
+        skin = 0.3
+        epsilon = 1.
+        sigma  = 0.
+
+        # system set up
+        system         = espressopp.System()
+        system.rng     = espressopp.esutil.RNG()
+        system.bc      = espressopp.bc.OrthorhombicBC(system.rng, box)
+        system.skin    = skin
+        nodeGrid       = espressopp.tools.decomp.nodeGrid(espressopp.MPI.COMM_WORLD.size)
+        cellGrid       = espressopp.tools.decomp.cellGrid(box, nodeGrid, rc, skin)
+        system.storage = espressopp.storage.DomainDecomposition(system, nodeGrid, cellGrid)
+
+        # interaction
+        interaction    = espressopp.interaction.VerletListLennardJones(espressopp.VerletList(system, cutoff=rc))
+        potLJ          = espressopp.interaction.LennardJones(epsilon, sigma, rc)
+        interaction.setPotential(type1=0, type2=0, potential=potLJ)
+        system.addInteraction(interaction)
+        
+        # integrator
+        integrator     = espressopp.integrator.VelocityVerlet(system)
+        integrator.dt = 0.001
+
+        # thermostat
+        thermostat     = espressopp.integrator.LangevinThermostat(system)
+        thermostat.gamma  = 1.0
+        thermostat.temperature = temperature
+        integrator.addExtension(thermostat)
+
+        # make dense system
+        global num_particles
+        particle_list = []
+        mass = 1.
+        for k in range(num_particles):
+            pid  = k + 1
+            pos  = system.bc.getRandomPos()
+            v    = Real3D(0,0,0)
+            type = 0
+            part = [pid, pos, type, v, mass]
+            particle_list.append(part)
+        system.storage.addParticles(particle_list, 'id', 'pos', 'type', 'v', 'mass')
+        system.storage.decompose()
+
+        print "Warm up. Sigma will be increased from 0. to 1."
```

Comment:
This warmup you could probably do once in setUpClass and in setUp load only coordinates from some file